cluster-operator/image.yaml

```diff
@@ -14,6 +14,12 @@ labels:
     value: "AMQ Streams Cluster Operator"
   - name: "io.openshift.tags"
     value: "messaging,amq,jboss"
+  - name: "licenses"
+    value: "/root/licenses"
+
+envs:
+  - name: "COM_REDHAT_COMPONENT"
+    value: "amqstreams11-clusteroperator-openshift-container"
```

Comment:
A couple of questions ...
What is this component? Does it contain some license files? From where it come from?
Another one ...
Isn't the `envs` section in the image YAML file used for passing env vars to the container image, while we are using it for passing a variable to the module installation script?

Reply:
@kyguy Assuming this is used only for the licences, I think we can call this just something like `amqstreams11`. I do not think we need to add the container name into it.

Reply:
@ppatierno Component is the brew package name used by OSBS to uniquely identify images. It comes from our dist-git repository. Cekit allows use to use the envs section from the image descriptor file at build time. 

@scholzj You are right, this is just for the licenses. I don't think these licensing requirements were written with multi-component products in mind. I agree your suggestion would make the most sense as it would be a lot cleaner. However, I don't think it would follow the license requirements document verbatim:

> $PRODUCT_IDENTIFIER will be the value of the container's com.redhat.component (former BZComponent) value

That being said, I am happy to veer from these exact requirements in favor for your suggestion which would be the most sensible. What do you think?

Reply:
I'm actually fine with both TBH. So I'm fine to leave the identifier you have there as well.